src/main/java/build/buildfarm/backplane/Backplane.java

```diff
@@ -120,6 +120,7 @@ ScanResult<String> scanCorrelatedInvocationIndexEntries(String cursor, int count
   /** Returns a set of the names of all active storage workers. */
   Set<String> getStorageWorkers() throws IOException;
 
+  // TODO this is just a namespace, but kind of jank for just digest function as a string...
```

Comment:
Does this go here? What's the namespace?

Reply:
3am venting. This was going to look much more complicated, and the namespace is (iirc) the action result space mapping by digest key. Ended up being exactly the same, likely due to buildfarm Digest message.